sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/administration/DocumentModelAdministrationClientBuilder.java

```diff
@@ -351,4 +362,15 @@ public DocumentModelAdministrationClientBuilder serviceVersion(DocumentAnalysisS
         this.version = version;
         return this;
     }
+
+    /**
+     * Sets the audience for the Azure Form Recognizer service.
```

Comment:
Update the javadoc to specify what is the default audience used if this is not specified.